charts/estafette-k8s-hpa-scaler/templates/_helpers.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "labels" }}
+labels:
+  app: {{.Chart.Name | quote}}
+  version: {{.Chart.Version | quote}}
```

Comment:
Not sure whether you want `version` and `appVersion` on each resource. Up until now we only set those in the `podSpec` so pods get these labels and they show up in Prometheus.

Reply:
Ok, I'll refactor these labels :+1: